tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -0,0 +1,191 @@
+import { VALIDATION_ERROR } from 'const';
+import { CONTENT_SCRIPT_NOT_FOUND, CONTENT_SCRIPT_EMPTY } from 'messages/javascript';
+import JavaScriptScanner from 'scanners/javascript';
+
+describe('content_scripts_file_absent', () => {
+  it('should show an error when content script is missing', () => {
+    const code = `browser.tabs.executeScript({file: '/content_scripts/absentFile.js'});`;
+    const fileRequiresContentScript = 'file-requires-content-script.js';
+    const jsScanner = new JavaScriptScanner(code, fileRequiresContentScript, {
+      existingFiles: {
+        fileRequiresContentScript,
```

Comment:
I guess these existingFiles properties should be `{ [fileRequiresContentScript]: '' }` (using the [ES2015 computed property name syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015), which would be equivalent to `{ 'file-requires-content-script.js': '' }`), am I right?